docs/postgres.md

```diff
@@ -88,6 +88,14 @@ dropdb opencollective_test
 dropuser opencollective
 ```
 
+## Migrations
+
+When creating migrations and interacting with the database please follow the guidelines below; 
+
+https://docs.opencollective.com/help/contributing/development/postgres-database
+
+**Note:** We use [Sequelize](https://sequelize.org/). Therefore, it's important to create the migrations with `npm run db:migration:create` so that it aligns with the default Sequelize file naming conventions. 
```

Comment:
Maybe we can write it with more specific usage, WDYT?

```suggestion
**Note:** We use [Sequelize](https://sequelize.org/). Therefore, it's important to create the migrations with `npm run db:migration:create -- --name <migration-name>` so that it aligns with the default Sequelize file naming conventions. 
```

Reply:
Yes good idea, I missed that part. 👍🏽